content/metadata.yaml

```diff
@@ -504,6 +504,19 @@ authors:
       string: "None"
       lastapproved: !!str 2020-04-30
     funders: NHGRI R01 HG10067
+  -
+    github: yemarshall
+    name: Yael Evelyn Marshall
+    initials: YEM
+    orcid: 0000-0003-3723-9683
+    contributions:
+      - Writing - Original Draft
+      - Writing - Review & Editing
+    code of conduct:
```

Comment:
@yemarshall I recommend we list something for `affiliations`.  If you do not have an active university or other institutional affiliation, would "Independent" be suitable?  I'm not exactly sure what the common convention is.

Reply:
The directions say for affiliation "strongly suggested" but not "mandatory". I don't have an exact answer right now b/c I have been contributing to this on top of / separately from my other research (I consider it part of my ~moonlighting volunteer / ~extracurricular research -- what many researchers do during current/new research challenges/large epidemics & now even more because lots of labs have been closed), I did not receive funding specifically towards this contribution although of course I am using my previous training, & much of the funding for the core of this project is through the central organizing researchers (not me, and in addition to everyone else who is contributing additional time on top of their separate research). So that we can go on with the other priorities, is it possible to delay filling this in until I can give you a better answer?

Reply:
Revisiting this later seems fine to me.  Skipping the affiliation doesn't break the Manubot build.

I'll let @rando2 make the final call before merging.

Reply:
alright, thanks :-)

Reply:
@agitter I don't know the least messy way to edit the file again (don't want to mess up the PR that was already approved), so asking you to add if you want affiliations (although not required, but just for regularity with other names) / you can put the following (without department because I had 2 programs funding me and the PhD part is done):
affiliations:
  - freelance copyeditor
  - recent PhD, Princeton University